bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/VisitDocumentController.java

```diff
@@ -29,6 +30,7 @@
 
 @Controller
 public class VisitDocumentController extends BaseRestController {
+    private static final String INSUFFICIENT_PRIVILEGE = "Insufficient privilege";
     private static final String INVALID_USER_PRIVILEGE = "User [%d] does not have require to delete patient file [%s]";
```

Comment:
I think there is a typo here (missing word privilege): "User [%d] does not have the required privilege to delete patient file [%s]". 

Also - is there a need to print the patient file name? The privilege is not based on file name but is an overall privilege to "delete files"? Such a specific message makes it appear that user needs specific privilege for the file.

Reply:
corrected typo. we are not returning the file name, but we are logging the error. I think its ok to leave it at that. we are logging for the beneffit the support team in case of insufficient privilege (or an potential attach). the filename can provide clue whether it was indeed legitimate or in case someone passing a relative path a potential attaack

Reply:
Got it. thanks!